build.gradle.kts

```diff
@@ -87,13 +89,29 @@ tasks.withType<Jar>().configureEach {
 
 application.mainClassName = "org.gradle.profiler.Main"
 
+node {
+    download = true
+}
+
+val generateHtmlReportJavaScript = tasks.register<NpxTask>("generateHtmlReportJavaScript") {
+    dependsOn("npmInstall")
+    val source = file("src/main/js/org/gradle/profiler/report/report.js")
+    val outputDir = layout.buildDirectory.dir("html-report")
+    val output = outputDir.map { it.file("org/gradle/profiler/report/report.js") }
+    inputs.file(source)
+    outputs.dir(outputDir)
+    command = "browserify@16.5.2"
+    setArgs(listOf(source.absolutePath, "--outfile", output.get().asFile))
+}
+
 tasks.processResources {
     into("META-INF/jars") {
         from(profilerPlugins) {
             // Removing the version from the JARs here, since they are referenced by name in production code.
             rename("""(.*)-\d\.\d.*\.jar""", "${'$'}1.jar")
         }
     }
+    from(generateHtmlReportJavaScript.map { it.outputs })
```

Comment:
Can't you use `from(generateHtmlReportJavaScript)` directly?

Reply:
Good point, done.